src/sqlittaaja/__init__.py

```diff
@@ -37,11 +39,16 @@ def main():
                 answer_dump = list(db.iterdump())
                 # Compare results between student's answer and the correct one.
                 if answer_rows == correct_rows and answer_dump == correct_dump:
-                    print(f"{student_name} +1")
+                    student_scores[info[0]] = student_scores.get(
```

Comment:
Use `student_name` here and down below. `info[0]` is a tupple of `(student_name, id)` and thus `info[0][0]` is the `student_name`.

Reply:
ID propably shouldn't be used as it relates to the exercise and not the student itself (I think). We could later use those IDs here when checking multiple exercises.